src/cloudShell.ts

```diff
@@ -178,7 +178,7 @@ export class AzureCloudShell extends BaseShell {
         const response: MessageItem = await vscode.window.showWarningMessage(message, DialogOption.ok, DialogOption.cancel);
```

Comment:
Skip showing the dialog, and add a note or todo comment to enable the dialog in the future.